src/components/icons/todoIcon.jsx

```diff
@@ -0,0 +1,27 @@
+import { SvgIcon } from "@mui/material";
+
+const ToDoIcon = (props) => {
+    return (
+        <SvgIcon
+            {...props}
+            viewBox="0 0 600 200"
+            style={{
+                width: "200px",
+                height: "90px",
+                display: "flex",
+                justifyContent: "center",
+                alignItems: "center",
+                margin: "auto" // Centers the icon horizontally in its container
+            }}
+        >
+            <rect x="50" y="50" width="500" height="100" rx="20" ry="20"
+                style={{ fill: "none", stroke: "#4CAF50", strokeWidth: 10 }} />
+            <text x="300" y="110" fontSize="100" fontFamily="Arial, sans-serif" fill="#4CAF50" textAnchor="middle"
+                dominantBaseline="middle">✓</text>
+            <text x="300" y="190" fontSize="50" fontFamily="Arial, sans-serif" fill="#333333" textAnchor="middle"
+                dominantBaseline="middle">To-Do List</text>
+        </SvgIcon>
```

Comment:
use prebuilt icons if you can

Reply:
It is something that the chat created for me, does it really matter ? It is very minor change, has nothing to do with the ui itself...

Reply:
in prisma we are mostly using mui icons, try to pick the closest one to the one you'd created and replace it

Reply:
Ok

Reply:
why create a component for icon? use mui <Icon /> or make the component generic